lib/measures/CreateTypicalBuilding/README.md.erb

```diff
@@ -0,0 +1,45 @@
+<%#= README.md.erb is used to auto-generate README.md. %>
```

Comment:
@jugonzal07 - Is this file needed?

Reply:
Nope-- it was just one of the files that gets autogenerated when creating measures from the application. Removed.